crates/control-interface/src/generated/ctliface.rs

```diff
@@ -205,3 +206,30 @@ pub struct ProviderDescription {
     #[serde(rename = "image_ref")]
     pub image_ref: Option<String>,
 }
+
+/// The standard function for serializing codec structs into a format that can be
+/// used for message exchange between actor and host. Use of any other function to
```

Comment:
I appreciate this comment 😄 Good to distinguish here the proper use of serialize

Reply:
This comment actually comes "out of the box" when you use the wapc code generator ;)